community/planning/roadmap.md

```diff
@@ -25,7 +25,7 @@ in some form.
 | Feature | Status | RFC | Issues | Documentation |
 | ------- | ------ | --- | ------- | ------------- |
 | Batch Tracking | *Under Development* | - | ["integration"](https://github.com/orgs/hyperledger/projects/1?card_filter_query=label%3A%22epic%3A+integration+component%22) | [Submitter Design]({% link community/planning/batch_submitter.md %}) |
-| Integration REST API | *Under Development* | - | ["integration"](https://github.com/orgs/hyperledger/projects/1?card_filter_query=label%3A%22epic%3A+integration+component%22) | - |
+| Integration REST API | *Under Development* | - | ["integration"]({% link community/planning/griddle/index.md %}) | - |
```

Comment:
Oh you need to update the next `-` the link you removed links the issues with the "integration" tag for the "issues" column